reverted commit
https://github.com/qemu/qemu/commit/e69b2c679030300023e4261633537c86a0ddaee9

to build both qemu-img and qemu-aarch64

[1536/1536] Linking target qemu-aarch64
FAILED: qemu-aarch64 
qdev.c:(.text+0x6c8): undefined reference to `qapi_event_send_device_deleted'

--- b/stubs/meson.build
+++ a/stubs/meson.build
@@ -55,12 +55,7 @@
 if have_user
   # Symbols that are used by hw/core.
   stub_ss.add(files('cpu-synchronize-state.c'))
+  stub_ss.add(files('qdev.c'))
-
-  # Stubs for QAPI events.  Those can always be included in the build, but
-  # they are not built at all for --disable-system --disable-tools builds.
-  if not (have_system or have_tools)
-    stub_ss.add(files('qdev.c'))
-  endif
 endif
 
 if have_system
